Example/SnapshotTests/SnapshotTestingTests.swift

```diff
@@ -28,15 +28,19 @@ final class SnapshotTestingTests: XCTestCase {
     #if swift(>=5.1) && canImport(SwiftUI)
```

Comment:
Do you want to remove this macro check from this test since you removed it from the source file?

Reply:
Hmm yeah I think we can probably remove it now. When we added it Swift 5.1 was new, but now all of our supported Xcode versions are >5.1.